core/renamer.ml

```diff
@@ -213,11 +215,12 @@ let renamer qs_from qs_to =
 let rename_function_definition : function_definition -> function_definition =
   fun { fun_binder
       ; fun_linearity
-      ; fun_definition = (tyvars_from, (pats, body))
+      ; fun_definition = (tyvars_from, f)
       ; fun_location
       ; fun_signature
       ; fun_frozen
       ; fun_unsafe_signature } ->
+  let (pats, body) = match f with | NormalFunlit (ps, bd) -> (ps, bd) | MatchFunlit (_,_) -> assert false in
```

Comment:
Please split this over multiple lines...
```suggestion
  let (pats, body) = 
     match f with 
     | NormalFunlit (ps, bd) -> (ps, bd) 
     | MatchFunlit (_,_) -> assert false 
  in
```
... or define and use a suitable extraction function, e.g.
```ocaml
let normal_funlit = function
   | NormalFunlit def -> def
   | _ -> assert false
```